src/sdk/account/toNexusAccount.ts

```diff
@@ -113,6 +113,9 @@ export type NexusSmartAccountImplementation = SmartAccountImplementation<
     encodeExecute: (call: Call) => Promise<Hex>
     encodeExecuteBatch: (calls: readonly Call[]) => Promise<Hex>
     getUserOpHash: (userOp: Partial<UserOperationStruct>) => Promise<Hex>
+    setActiveValidationModule: (validationModule: BaseValidationModule) => void
+    getActiveValidationModule: () => BaseValidationModule
```

Comment:
Wanna try and keep this file as clean & lean as possible as I we'll potentially copy / pasting into viem.

Reply:
Yes there should be a method to fetch the active validation module on a smart account through SDK. 
It's an easy to check what validation module you are going to use before running the next user operation.
Sure I can remove the getter and add the activeModule as an extension, if viem has any issue with this we'll remove it.

Reply:
btw I noticed (if we still have similar method like sendTransaction or sendUserop) in module sdk that it passes activeModule instance as a param. so you would not need to "set". both can have its own benefits

Reply:
We can add in another PR a param "validationModule" on sendUserOperation and sendTransaction, when this param is passed we use it instead of default active module